### PR TITLE
facets: fix 0 results problem in all collections

### PR DESCRIPTION
**Pull request by ioannistsanaktsidis**
Signed-off-by: Ioannis Tsanaktsidis <ioannis.tsanaktsidis@cern.ch>

### CODE REVIEW

File: inspirehep/base/searchext/mappings/institutions.json [hunk 1]
```diff
@@ -102,6 +102,14 @@
                 "ICN": {
                     "type": "string",
                     "copy_to": ["affautocomplete"]
+                },
+                "address": {
```
Comment (jmartinm) [inline inspirehep/base/searchext/mappings/institutions.json]: Can you check if it is possible to use `fields` in order to have the `address` field analyzed and a `raw` version not analyzed? And for the facets we will use `address.raw`

**Comment by ioannistsanaktsidis** [comment]
I tried to exactaddress and seems to working fine. We can keep it like this I guess.

File: inspirehep/base/searchext/mappings/institutions.json [hunk 2]
```diff
@@ -102,6 +102,23 @@
...
+                        }
+                    }
+                },
+                "address": {
+                    "properties": {
+                        "country": {
+                            "type": "string",
+                            "copy_to": ["exactaddress"]
+                        }
+                    }
```
Comment (jmartinm) [inline inspirehep/base/searchext/mappings/institutions.json]:
```
"address": {
    "properties": {
        "country": {
             "type": "string",
             "fields": {
                 "raw": { 
                     "type":  "string",
                    "index": "not_analyzed"
                }
            }
     }
}
```

And then you would have `address.country` and `address.country.raw`

**Comment by ioannistsanaktsidis** [comment]
Updated!

**Comment by jmartinm** [comment]
After merging https://github.com/inspirehep/inspire-next/commit/dd847d83c46e646248e21438fc1cf61b2ba6d2c5 the keywords that are in the ElasticSearch mapping should be discovered automatically so we might not need to add most of keywords in `SEARCH_ELASTIC_KEYWORD_MAPPING`

Can you rebase and which ones we can remove? Once it is working with latest `master` I can merge this.

**Comment by ioannistsanaktsidis** [comment]
@jmartinm I updated the PR. We can only remove the series keyword. I tried to remove the others too, but the facets  not working properly without them.

**Comment by jmartinm** [comment]
> @jmartinm I updated the PR. We can only remove the series keyword. I tried to remove the others too, but the facets not working properly without them.

@Panos512 isn't it strange? I would assume now that mappings are taken automatically doing something like `"continent": ["continent"]` should not be needed. Can you take install this PR and see if you get the same problem?

**Comment by ioannistsanaktsidis** [comment]
@jmartinm  Updated!

**Comment by ioannistsanaktsidis** [comment]
@jmartinm  we solved the problem with @Panos512 . The problem was that in my local the query-parser
and the invenio-search  were not updated. Cheers!

**Comment by ioannistsanaktsidis** [comment]
@jmartinm  this one is also ready.

**Pull request merged**